db/migrate/20220812194320_create_comment.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :authorId
+      t.string :postId
```

Comment:
- [ ] Instead of add Columns `authorId` and `postId` to the table `comments` without an index, I'll suggest you implement them as below so that you'll have a reference, then a foreign-key.
```ruby
 create_table :comments do |t|
    t.text :text
    t.timestamps
    t.references :author, foreign_key: { to_table: 'users' } # this will create author_id and a corresponding index
    t.references :post, foreign_key: true # this will create post_id and a corresponding index
  end
```
Also, if you want you can implement it using the command of adding a reference column to a table by using a command similar to this: `rails generate migration AddUsersRefToComment user:references`. Read this [article](https://riptutorial.com/ruby-on-rails/example/3492/add-a-reference-column-to-a-table) for more information.
**Kindly, do this for all tables which need a reference column, `posts`, `comments` and `likes` tables.**

Reply:
@RolandM99 Noted. Added foreign keys via this command for all tables 
```
 rails generate migration AddUserRefToPost posts:references